docs/modules/ROOT/pages/tmail-backend/webadmin.adoc

```diff
@@ -131,4 +131,305 @@ Returns the list of Team Mailboxes `bob@domain.tld` user has access to :
 Return codes:
 
 - `400` Invalid user
-- `404` User not found
\ No newline at end of file
+- `404` User not found
+
+== Rate limiting
+
+=== Create a new plan
+
+....
+curl -XPOST http://ip:port/rate-limit-plans/{RateLimitingPlanName}
+ -H "Content-Type: application/json"
+ -d '{
+	"transitLimits": [{
+			"name": "receivedMailsPerHour",
+			"periodInSeconds": 3600,
+			"count": 100,
+			"size": 2048
+		},
+		{
+			"name": "receivedMailsPerDay",
+			"periodInSeconds": 86400,
+			"count": 1000,
+			"size": 4096
+		}
+	],
+	"relayLimits": [{
+		"name": "relayMailsPerHour",
+		"periodInSeconds": 3600,
+		"count": 100,
+		"size": 2048
+	}],
+	"deliveryLimits": [{
+		"name": "deliveryMailsPerHour",
+		"periodInSeconds": 3600,
+		"count": 100,
+		"size": 2048
+	}]
+}'
+....
+
+Return a created plan:
+....
+{
+	"planId": "6f2f0f9b-7b97-4f26-b640-a250512128df",
+	"planName": "planName1",
+	"transitLimits": [{
+			"name": "receivedMailsPerHour",
+			"periodInSeconds": 3600,
+			"count": 100,
+			"size": 2048
+		},
+		{
+			"name": "receivedMailsPerDay",
+			"periodInSeconds": 86400,
+			"count": 1000,
```

Comment:
I think we don't need to return the plan that was created... A 201 CREATED should be enough...

Reply:
I think we can return just planId.